server/src/main/java/org/elasticsearch/monitor/os/OsProbe.java

```diff
@@ -567,22 +569,33 @@ private String getPrettyName() throws IOException {
     }
 
     /**
-     * The lines from {@code /etc/os-release} or {@code /usr/lib/os-release} as a fallback. These file represents identification of the
-     * underlying operating system. The structure of the file is newlines of key-value pairs of shell-compatible variable assignments.
+     * The lines from {@code /etc/os-release} or {@code /usr/lib/os-release} as a fallback, with an additional fallback to
+     * {@code /etc/system-release}. These files represents identification of the underlying operating system. The structure of the file is
```

Comment:
nanonit: s/represents/represent/ 
:)

Reply:
I pushed c09423d9013165cadf5a4d2f4223d86e10c2f6c3.

Reply:
Thanks for double-checking. 😄